hdr/sqlite_modern_cpp/type_wrapper.h

```diff
@@ -4,7 +4,20 @@
 #include <string>
 #include <memory>
 #include <vector>
-
+#ifdef __has_include
+#if __cplusplus >= 201703 && __has_include(<string_view>)
+#define MODERN_SQLITE_STRINGVIEW_SUPPORT
+#include <string_view>
+#define STR_REF std::string_view
+#define U16STR_REF std::u16string_view
+#else
+#define STR_REF std::string
+#define U16STR_REF std::u16string
+#endif
+#else
+#define STR_REF std::string
+#define U16STR_REF std::u16string
+#endif
```

Comment:
It is more modern and more robust to use typedefs instead of macros. It is also more consistant with `optional` etc.